stim_selective_stop_signal/experiment.js

```diff
@@ -646,6 +703,12 @@ var practice_node = {
         practice_feedback_text +=
           '</p><p class = block-text>Your accuracy is too low. Remember, the correct keys are as follows: ' + prompt_text
       }
+      
+      if (ignoreRespond_percent >= .4){
+      	selective_feedback_text +=
```

Comment:
If you want this feedback (and you want subjects to conform to it) you have to add it to the conditional at [line 675](https://github.com/jkl071/expfactory-experiments/blob/5fe2bd16084ff2b74681436115115be62a252ad2/stim_selective_stop_signal/experiment.js#L675)